pipeline/matcher/src/test/scala/uk/ac/wellcome/platform/matcher/matcher/WorkMatcherTest.scala

```diff
@@ -64,7 +64,7 @@ class WorkMatcherTest
     }
   }
 
-  it("doesn't store an invisible work and sends the work id") {
+  it("stores an invisible work and sends the work id") {
```

Comment:
:thinking: do we  want to store all invisible works? What are the repercussions  of that?

Reply:
I think we have to. The matcher stores the nodes of the work graph in the database and then it calculates the connected components on that graph.
All works created by the mets transformer will be invisible works. For the matcher to be able to group them to the physical sierra work it has to store them as all the other works. 
We will end up storing all the works that are currently invicible for other reasons, but I don't think this will have a major impact. 
The merger won't link them and the output in the api won't change.
Performance and price wise I don't think we might end up spending a bit more because we increase operations but I don't think it will be much

Reply:
:thinking: Just a thought, do you think it would make sense for there to be another type separate to `InvisibleWork`? Me and @jamesgorrie discussed at one point whether `PartialWork` or something like that would be a better description for this use case.

Beyond performance / price, the thing I am more worried about is introducing subtle bugs some how (as we are potentially using invisible to represent two quite separate things).

Reply:
I'm not sure it's a separate use case though. Non mets Invisible works are still partial and they are invisible because they don't have enough information to be shown in the api. 
I'm not sure whether adding another type would simplify our life or complicate it

Reply:
:+1: ok let's just stick with this for now, we can always split off later